### PR TITLE
[dom-gpu] XALT cce/9.0.2 possible workaround

### PR DESCRIPTION
**Pull request by victorusu**
This is a possible workaround, but not complete

### CODE REVIEW

**Comment by ekouts** [comment]
Hi @victorusu , is the GROMACS recipe in this PR by mistake? Also, should the tag in the title be [dom]?

**Comment by victorusu** [comment]
@ekouts, yes it is...
I just merged from master and it has disappeared.

File: easybuild/easyconfigs/x/xalt/xalt-2.8.eb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+source_urls = ['https://github.com/xalt/xalt/archive/']
+#source_urls = ['https://github.com/victorusu/xalt.git']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = [('Daint_config.py', 'Config/')]
+
+local_transmission = "curl"
+
+# generic configurations
+preconfigopts  = 'cd %(builddir)s && cp xalt-xalt-%(version)s/Config/Daint_config.py . && '
```
Comment (vkarak) [inline easybuild/easyconfigs/x/xalt/xalt-2.8.eb:24]: This `xalt-xalt` name is a bit strange.

File: easybuild/easyconfigs/x/xalt/xalt-2.8.eb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+preconfigopts += "rm -rf xalt-xalt-%(version)s && "
+preconfigopts += 'git clone https://github.com/victorusu/xalt.git xalt-xalt-%(version)s && '
```
Comment (vkarak) [inline easybuild/easyconfigs/x/xalt/xalt-2.8.eb:26]: Why not putting that in `source_urls`?

File: easybuild/easyconfigs/x/xalt/xalt-2.8.eb [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# contributed by Victor Holanda (CSCS)
+
+name = 'xalt'
+version = "2.8"
```
Comment (vkarak) [inline easybuild/easyconfigs/x/xalt/xalt-2.8.eb:4]: I would suggest adding a suffix to make sure that this is our patched version.

File: easybuild/easyconfigs/x/xalt/xalt-2.8.eb (hunk 2 above)
Comment (victorusu) [reply easybuild/easyconfigs/x/xalt/xalt-2.8.eb:26]: This is out-dated

File: easybuild/easyconfigs/x/xalt/xalt-2.8.eb (hunk 3 above)
Comment (victorusu) [reply easybuild/easyconfigs/x/xalt/xalt-2.8.eb:4]: I am not sure if we should use a suffix, due to the fact the xalt installation is already daint/dom directed and the patch is almost the same as of `Config.py` patch that we apply to get the config. This is just an additional config.

Comment (victorusu) [reply easybuild/easyconfigs/x/xalt/xalt-2.8.eb:4]: @vkarak, what do you think?

Comment (vkarak) [reply easybuild/easyconfigs/x/xalt/xalt-2.8.eb:4]: Fine for me.

**Pull request merged**